x/bank/internal/keeper/keeper.go

```diff
@@ -156,6 +147,18 @@ func (keeper BaseSendKeeper) SendCoins(
 	if !amt.IsValid() {
 		return sdk.ErrInvalidCoins(amt.String())
 	}
+
+	ctx.EventManager().EmitEvents(sdk.Events{
```

Comment:
`Emit` a bit confusing here although it makes sense... Does `AddEvents` not make sense?